wiki/Tournaments/osu!_at_UCSD/TGEX_2022/en.md

```diff
@@ -11,19 +11,311 @@ tags:
+!["Welcome to TGEX" pixel art banner from the main spreadsheet](img/spreadsheet-pixel-art.png)
 
-[osu! at UCSD](/wiki/Community/Organisations/osu!_at_UCSD) is hosting an osu! LAN tournament at **Triton Gaming Expo 2022** (***TGEX 2022***) in collaboration with *Triton Gaming*. Attendance and tournament participation is open to everybody. The exact schedule and details for the tournament have not been released yet.
+**Triton Gaming Expo 2022** (***TGEX 2022***) was an osu! BYOC LAN event hosted by [osu! at UCSD](/wiki/Community/Organisations/osu!_at_UCSD) in collaboration with *Triton Gaming*. It shared its name with its parent event, a gaming convention spanning many more games and activities. This event and tournament was osu! at UCSD's fourth appearance at TGEX.
 
-## TGEX details
+## TGEX
 
-TGEX will take place on Sunday, May 29th at [Price Center](https://en.wikipedia.org/wiki/Price_Center) at the University of California, San Diego (*UCSD*). The event is expected to run from 12:00 p.m. to 10:00 p.m. (UTC-7).
+![TGEX banner art](img/tgex-banner.png)
 
-Some parts of the event may require checking in at Ballroom West (rooms 215 and 216). There is no entry fee, but proof of vaccination against COVID-19 may be required.
+TGEX 2022 took place on Sunday, May 29th at [Price Center](https://en.wikipedia.org/wiki/Price_Center) at the [University of California, San Diego](https://en.wikipedia.org/wiki/University_of_California,_San_Diego) (*UCSD*). The main floor was open from 12:00 p.m. to 10:00 p.m., but each event inside had its own schedule. The osu! LAN ran from around 11:15 a.m. to 9:00 p.m. in Earl Warren College Room.
 
-Besides the osu! tournament, TGEX is hosting tournaments for many other games, panels with game industry experts, an artist alley, cosplay gathering, keyboard showcase, and more. More details can be found in the [Triton Gaming Discord server](https://discord.gg/tritongaming).
+Aside from the osu! event, TGEX hosted tournaments for many other games, panels with game industry experts, an artist alley, cosplay gathering, keyboard showcase, and more. More details can be found in the [Triton Gaming Discord server](https://discord.gg/tritongaming).
 
 ## Links
 
+- [Main spreadsheet](https://docs.google.com/spreadsheets/d/1eckyCyhWb1Y2bNuopxzE-C-XSJ4Qil0t5utQykW3vTA/edit)
+- [Challonge bracket](https://challonge.com/yqvqkyfq)
+- [Seeding results spreadsheet](https://docs.google.com/spreadsheets/d/1WZbENIjcmtDUcwqj-heN7MfDbsBZ8znvfprOP-Sve0g/edit)
 - [osu! at UCSD Discord server](https://discord.gg/DqBVmv5Knx)
 - [Triton Gaming Discord server](https://discord.gg/tritongaming)
+
+## Tournament format
+
+The tournament hosted at TGEX 2022 was a 1v1 double-elimination tournament with a seeding round before the bracket. More details and rules can be found in the ["Info & Rules" tab of the main spreadsheet](https://docs.google.com/spreadsheets/d/1eckyCyhWb1Y2bNuopxzE-C-XSJ4Qil0t5utQykW3vTA/edit#gid=1843073213).
```

Comment:
also part of the [template](https://github.com/ppy/osu-wiki/blob/master/wiki/Tournaments/TEMPLATE.md#title) to specify the game mode

```suggestion
The tournament hosted at TGEX 2022 was a 1v1 double-elimination osu! tournament with a seeding round before the bracket. More details and rules can be found in the ["Info & Rules" tab of the main spreadsheet](https://docs.google.com/spreadsheets/d/1eckyCyhWb1Y2bNuopxzE-C-XSJ4Qil0t5utQykW3vTA/edit#gid=1843073213).
```